src/helpers/figures/charts/geo_chart.ts

```diff
@@ -38,13 +38,13 @@ import {
 } from "./chart_common";
 import { CHART_COMMON_OPTIONS } from "./chart_ui_common";
 import {
-  getChartLayout,
   getChartTitle,
   getGeoChartData,
   getGeoChartDatasets,
   getGeoChartScales,
   getGeoChartTooltip,
 } from "./runtime";
+import { getChartLayout } from "./runtime/chartjs_layout";
```

Comment:
Strange import change ? I'm guessing your IDE did some strange things ? Same in pyramid_chart.ts

Reply:
In fact, I first started to remove this `getChartLayout` method in this PR some weeks ago, but yesterday, when I rebased, I saw that it now takes an argument so it was now needed as a function and not a constant value, so I had to put this method back and reimport it